Net02_1/Catalog.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Net02_1
+{
+    internal class Catalog : IEnumerable
+    {
+        List<Book> _books;
+
+        public Catalog()
+        {
+            _books = new List<Book>();
+        }
+
```

Comment:
What's happened if you try to add already existing book in a catalog?

Reply:
creating a duplicate. I fixed it